src/v2/coins/btg.js

```diff
@@ -26,6 +26,14 @@ class Btg extends Btc {
     return false;
   }
 
+  supportsP2shP2wsh() {
```

Comment:
How did we previously determine whether a coin had wrapped segwit support? o.0

Reply:
We didn't.